<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-08T22:59:30 IST" name="Default suite" finished-at="2023-08-08T22:59:32 IST" duration-ms="1145">
    <groups>
    </groups>
    <test started-at="2023-08-08T22:59:30 IST" name="Default test" finished-at="2023-08-08T22:59:32 IST" duration-ms="1145">
      <class name="testcases.firstTest">
        <test-method signature="loginTest(java.lang.String,java.lang.String)[pri:0, instance:testcases.firstTest@313ac989]" started-at="2023-08-08T22:59:30 IST" name="loginTest" data-provider="logindataxl" finished-at="2023-08-08T22:59:31 IST" duration-ms="1071" status="FAIL">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void testcases.firstTest.loginTest(java.lang.String,java.lang.String) throws java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: loginTest([java.lang.String arg0, java.lang.String arg1])
Arguments: [(java.lang.String) md50@amitknksckmnvkdyonline.com,(java.lang.String) arsnkd@321kndkd,(java.lang.String) md50@amitknksckmnvkdyonline.com,(java.lang.String) arsnkd@321kndkd]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void testcases.firstTest.loginTest(java.lang.String,java.lang.String) throws java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: loginTest([java.lang.String arg0, java.lang.String arg1])
Arguments: [(java.lang.String) md50@amitknksckmnvkdyonline.com,(java.lang.String) arsnkd@321kndkd,(java.lang.String) md50@amitknksckmnvkdyonline.com,(java.lang.String) arsnkd@321kndkd]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:918)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:37)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- testcases.firstTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
